website/docs/tutorials/08-interacting-with-starknet/07-multicall.md

```diff
@@ -0,0 +1,52 @@
+# Multicall
+
+## Overview
+Multicall allows executing multiple calls while providing [atomicity](https://en.wikipedia.org/wiki/Atomicity_(database_systems)). If one call fails, the entire operation is aborted. Use it to prevent leaving your system in an inconsistent state. 
+
+
+## Usage example
+
+Before you run [`protostar multicall`](/docs/cli-reference#multicall), you need to [create an account](./05-deploy-account.md) in order to [sign the transaction](./06-signing.md). If you want to deploy contract within a multicall, you need to [declare](./03-declare.md) it first.
+
+
+Then, create a toml file containing call [array of tables](https://toml.io/en/v1.0.0#array-of-tables).
```

Comment:
```suggestion
Then, create a toml file containing the multicall [array of tables](https://toml.io/en/v1.0.0#array-of-tables).
```

Reply:
"multicall array of tables" doesn't make sense. It's `[[call]]` not `[[multicall]]`.

Reply:
So it should be plural i suppose?

`containing [array_of_tables] including a sequence of calls

Reply:
`containing calls defined with [array of tables]`